docs/kyma/04-01-overview.md

```diff
@@ -5,30 +5,48 @@ type: Installation
 
 Kyma is a complex tool which consists of many different [components](#details-components) that provide various functionalities to extend your application. This entails high technical requirements that can influence your local development process. To meet the customer needs, we ensured Kyma modularity. This way you can decide not to include a given component in the Kyma installation, or install it after the Kyma installation process.
 
-To make the local development process easier, we introduced the **Kyma Lite** concept in which case some components are not included in the local installation process by default. These are the Kyma and Kyma Lite components:
+To make the local development process easier, we introduced the **Kyma Lite** concept in which case some components are not included in the local installation process by default. These are the Kyma and Kyma Lite components in their installation order:
 
 | Component | Kyma | Kyma Lite |
 |----------------|------|------|
-| `core` | ✅ | ✅ |
-| `rafter` | ✅ | ✅ |
 | `cluster-essentials` | ✅ | ✅ |
+| `testing` | ✅ | ✅ |
+| `istio-init` | ✅ | ✅ |
+| `istio` | ✅ | ✅ |
+| `xip-patch` | ✅ | ✅ |
+| `istio-kyma-patch` | ✅ | ✅ |
+| `tekton-pipelines` | ⛔️ | ⛔️ |
+| `knative-serving-init` | ✅ | ✅ |
+| `knative-serving` | ✅ | ✅ |
+| `knative-eventing` | ✅ | ✅ |
+| `knative-eventing-channel-kafka` | ⛔️ | ⛔️ |
+| `knative-eventing-channel-kafka-tests` | ⛔️ | ⛔️ |
+| `dex` | ✅ | ✅ |
+| `ory` | ✅ | ✅ |
+| `api-gateway` | ✅ | ✅ |
+| `function-controller-init` | ⛔️ | ⛔️ |
```

Comment:
`function-controller-init` and `function-controller` are installed after `knative-eventing-channel-kafka-tests`. Please update table :)

Reply:
Forget about this comment.

Reply:
As discussed, this order is already based on changes in this PR: https://github.com/kyma-project/kyma/pull/7244